### DISCUSSION TITLE
Advertisement of new, non-existing features in the UI of the application

### DISCUSSION THREAD

**Discussion by almereyda** [teableio/teable#850] [General]
When using Teable now, I'm also finding non-working UI elements, which had already been a nuisance for our user base with NocoDB. When using the application, they are presented with main navigation links to the Authorization Matrix and to Automations, which don't exist as of now.

Unfortunately our users do not use computers so much and it is hard for them to keep up with changing user interfaces. Now knowing that there are interface components with no obvious function, they are even less curious and more afraid to try things out on their own.

We would like to see this kind of conditioning disabled. Can we imagine an environmental variable or application setting, which allows to toggle "Tease upcoming application features".

What do you think?

**Comment by tea-artist** [comment]
I agree with you, buttons removed.

**Comment by almereyda** [comment]
Thank you!
